src/app/containers/RecentVideoEpisodes/index.jsx

```diff
@@ -102,8 +104,17 @@ const RecentVideoEpisodes = ({ episodes }) => {
             </EpisodeList.Link>
             {episode.episodeTitle && (
               <span role="text">
-                <EpisodeList.Metadata as="time">
-                  {formatDate(episode.timestamp)}
+                <EpisodeList.Metadata>
```

Comment:
I think `as` would work here and remove an unnecessary span eg:

`<EpisodeList.Metadata as={Timestamp} format="..." ... />